environment/src/test/java/com/sequenceiq/environment/environment/service/LoadBalancerPollerServiceTest.java

```diff
@@ -65,33 +66,35 @@ public class LoadBalancerPollerServiceTest {
 
     private final FlowEndpoint flowEndpoint = Mockito.mock(FlowEndpoint.class);
```

Comment:
Please eliminate the explicit `Mockito.mock()` invocations and add `@Mock` annotations instead.

Reply:
reworked